docs/cvms.md

```diff
@@ -0,0 +1,106 @@
+# Confidential Virtual Machines (CVMs)
+
+## Overview
+
+Confidential Virtual Machines (CVMs) provide a secure environment for running computations while ensuring data confidentiality. CVMs are available in multiple configurations, allowing users to select a flavor that best suits their needs. The currently supported flavors include:
+
+- **GCP**: Confidential Virtual Machines hosted on Google Cloud Platform.
+- **Azure**: Confidential Virtual Machines deployed on Microsoft Azure.
+- **Manager**: Confidential Virtual Machines running on our dedicated cloud infrastructure.
+
+Before computations can be executed, a CVM must be created and properly configured. Once set up, it can be selected for running workloads securely.
+
+## **Confidential Computing Virtual Machines (CVM) Configurations**  
+
+- **GCP CVMs** run on the **N2D machine series**, powered by **AMD EPYC Milan and Rome processors**. These machines support up to **224 vCPUs and 896 GB RAM**, offering high-bandwidth networking and flexible pricing options. They use **SEV-SNP** for memory encryption, ensuring data remains confidential even from the hypervisor. N2D instances support persistent and local SSD storage but do not support GPUs or nested virtualization.  
+
+- **Azure CVMs** utilize the **DCasv5 series**, featuring **AMD EPYC 7763v (Milan) processors** with **up to 96 vCPUs and 384 GB RAM**. These machines incorporate **SEV-SNP** for full memory encryption and hardware-isolated execution. They support premium remote storage but lack local disks, making them ideal for workloads requiring strict security without temporary storage dependencies.  
+
+- **Manager** runs on an **AMD EPYC 7443P (Milan) processor** with **24 cores and 30 GB RAM**, providing a secure and efficient environment for coordination tasks. It leverages **SEV-SNP** for full memory encryption, ensuring strong confidential computing capabilities. The system operates on **Ubuntu 24.04.1 LTS** and features **446.6 GB of storage**, with a **1 GB EFI partition and 445.6 GB root partition**. It supports virtualization via **AMD-V** and includes mitigations against speculative execution vulnerabilities, making it a reliable control node in confidential computing infrastructures.  
```

Comment:
rather than host info we need info about the cvms we create on the host like EOS and its capabilities, the nature of manager generated vms for example ssh being disabled, full memeory encryption, no disk on boot etc.